docs/installation/configuration.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Configuration Options"
+weight: 20
+---
+
+This page focuses on the options that can be used when you set up K3s for the first time:
+
+- [K3s Install Script](#configuration-with-install-script)
+- [K3s Binary](#configuration-with-binary)
+- [Configuration File](#configuration-file)
+
+For more advanced options, refer to [this page](/advanced).
+
+## Configuration with install script
+
+As mentioned in the [Quick-Start Guide](/quick-start), you can use the installation script available at https://get.k3s.io to install K3s as a service on systemd and openrc based systems.
+
+You can use a combination of `INSTALL_K3S_EXEC`, `K3S_` enviromental variables and command flags to configure the installation.
+
+To illustrate this, the following commands all result in the same behavior of registering a server without flannel and with a token:
+
+```bash
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--flannel-backend none --token 12345" sh -s -
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server --flannel-backend none" K3S_TOKEN=12345 sh -s -
+curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="server" sh -s - --flannel-backend none
+curl -sfL https://get.k3s.io | K3S_TOKEN=12345 sh -s - server --flannel-backend none
+curl -sfL https://get.k3s.io | sh -s - --flannel-backend none --token 12345
+```
+
+For details on all enviromental variables, see [Enviromental Variables.](/reference/env-variables)
```

Comment:
```suggestion
For details on all environmental variables, see [Environmental Variables.](/reference/env-variables)
```

Reply:
Converted to correct spelling of `environment` as that is the correct name.